content/influxdb/v2.2/write-data/no-code/scrape-data/manage-scrapers/create-a-scraper.md

```diff
@@ -10,6 +10,9 @@ menu:
     name: Create a scraper
     parent: Manage scrapers
 weight: 301
+related:
+  - /influxdb/v2.2/write-data/no-code/scrape-data/
+
 ---
 
 Create a new scraper in the InfluxDB user interface (UI).
```

Comment:
Let's define scraper at first mention. We could link to the glossary at first mention on applicable pages.
We could add an entry for InfluxDB scraper:
_InfluxDB scrapers collect data from specified targets at regular intervals, then write the scraped data to an InfluxDB bucket. Scrapers can collect data from any HTTP(S)-accessible endpoint that provides data in the [Prometheus data format](https://prometheus.io/docs/instrumenting/exposition_formats/)._

and then associate "InfluxDB scraper" with the entry "scrape."

Reply:
done